backend-api/template.yaml

```diff
@@ -1112,6 +1112,126 @@ Resources:
         - !Ref PermissionsBoundary
         - !Ref AWS::NoValue
 
+  WellKnown5XXLowThresholdWarningAlarm:    
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmDescription: "/.well-known/jwks.json endpoint has returned at least 2 5XX status codes twice in the past 5 minutes"
+      AlarmName: !Sub "${AWS::StackName}-LowThresholdWellKnown5XXApiGwAlarm"
+      AlarmActions:
+        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning"
+      OKActions:
+        - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:platform-alarms-sns-warning"
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      DatapointsToAlarm: 2
+      EvaluationPeriods: 5
+      TreatMissingData: notBreaching
+      Threshold: 2
+      Metrics:
+        - Id: errorThreshold
+          Label: errorThreshold
+          ReturnData: true
+          Expression: IF((invocations>4) && (errorSum5XX>2) && (errorPercentage5XX>1),errorPercentage5XX,0)
+        - Id: invocations
+          ReturnData: false
+          MetricStat:
+            Metric:
+              Namespace: AWS/ApiGateway
+              MetricName: Count
+              Dimensions:
+                - Name: Method
+                  Value: GET
+                - Name: ApiName
+                  Value: !Sub ${AWS::StackName}-public-api
```

Comment:
this API doesn't exist anymore - the name has changed

Reply:
Changed to `${AWS::StackName}-sessions-api`